plugins/action/shutdown.py

```diff
@@ -144,7 +144,10 @@ def get_shutdown_command(self, task_vars, distribution):
 
         full_path = [x['path'] for x in find_result['files']]
         if not full_path:
-            raise AnsibleError('Unable to find command "{0}" in search paths: {1}'.format(shutdown_bin, search_paths))
+            display.vvv('Unable to find command "{0}" in search paths: {1}, will attempt a shutdown using systemd '
+                        'directly '.format(shutdown_bin, search_paths))
+            self._shutdown_command = 'systemctl poweroff'
+            return self._shutdown_command
```

Comment:
Hmm, the action should also check whether `systemctl` is present, and only then use `systemctl poweroff`. I would probably also pass `systemctl` to https://docs.ansible.com/ansible/latest/collections/ansible/builtin/find_module.html#parameter-patterns in the find module call above; that would require the result checking to get more complicated though.